cves/CVE-2009-3548.yml

```diff
@@ -183,8 +196,12 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      By default, the admin password should at least be specified for some amount of security.
+      If a user just clicks through midlessly during installation, the blank password will persist.
+      This vulnerability defaulted it to blank. Perhaps this is not the user's main concern. Nonetheless,
+      not secure by default.
```

Comment:
I absolutely agree with this. This should be a default requirement. I wonder if the developer missed the requirement or if this was a result of laziness.